builder

```diff
@@ -83,6 +83,8 @@ mkdir -p "${rootfs_dir}/${kernel_header_dir}"
   rsync -a "${kernel_source_dir}/include/" "${rootfs_dir}/${kernel_header_dir}/include/"
 )
 
+find "${rootfs_dir}/${kernel_header_dir}" \( -name '.install' -or -name '..install.cmd' \) -delete
```

Comment:
To avoid accidents, does it worth checking if `"${rootfs_dir}/${kernel_header_dir}"` is resolved to `/`, if the variables are empty?

It should normally not happen, but if the user changes the config...